hedera-node/src/main/java/com/hedera/services/state/submerkle/ExpirableTxnRecord.java

```diff
@@ -358,27 +358,18 @@ public ExpirableTxnRecord copy() {
 
 	public static ExpirableTxnRecord fromGprc(TransactionRecord record) {
 		List<EntityId> tokens = NO_TOKENS;
-		List<CurrencyAdjustments> tokenAdjustments = null;
-		List<NftAdjustments> nftTokenAdjustments = null;
+		List<CurrencyAdjustments> tokenAdjustments = new ArrayList<>();
```

Comment:
These lists are purposely initialized as `null` instead of a new `ArrayList` because in many cases we can avoid creating these objects.

Reply:
Yeah, changed the implementation to keep these lists to still be initialized as `null`.